docs/troubleshooting.md

```diff
@@ -1,32 +1,54 @@
 # Troubleshooting
 
-- [Installation](#installation)
-  - [How to use LISA in WSL](#how-to-use-lisa-in-wsl)
-  - [Cannot find package after run `poetry install`](#cannot-find-package-after-run-poetry-install)
-  - [Error: Poetry could not find a pyproject.toml file](#error-poetry-could-not-find-a-pyprojecttoml-file)
-  - [Poetry related questions](#poetry-related-questions)
+- [Troubleshooting](#troubleshooting)
+  - [Installation](#installation)
+    - [How to use LISA in WSL](#how-to-use-lisa-in-wsl)
+    - [Cannot find package after run `poetry install`](#cannot-find-package-after-run-poetry-install)
+    - [Error: Poetry could not find a pyproject.toml file](#error-poetry-could-not-find-a-pyprojecttoml-file)
+  - [Using VSCode](#using-vscode)
+    - [Cannot find Python Interpreter by Poetry](#cannot-find-python-interpreter-by-poetry)
+    - [VSCode Python extension no longer supports `python.pythonPath` in `setting.json`](#vscode-python-extension-no-longer-supports-pythonpythonpath-in-settingjson)
   - [Other issues](#other-issues)
+    - [Poetry related questions](#poetry-related-questions)
 
 ## Installation
 
 ### How to use LISA in WSL
 
-If you are using WSL, installing Poetry on both Windows and WSL may cause both platforms' versions of Poetry to be on your path, as Windows binaries are mapped into `PATH` of WSL. This means that the WSL `poetry` binary _must_ appear in your `PATH` before the Windows version, or this error will appear:
+If you are using WSL, installing Poetry on both Windows and WSL may cause both platforms' versions of Poetry to be on your path, as Windows binaries are mapped into `PATH` of WSL. This means that the WSL `poetry` binary _must_ appear in your `PATH` before the Windows version, otherwise this error will appear:
 
 > `/usr/bin/env: ‘python\r’: No such file or directory`
 
 ### Cannot find package after run `poetry install`
 
-Poetry is case sensitive. When in windows, make sure the case of path is consistent every time.
+Poetry is case sensitive when reading the path. Please make sure the path is consistent all the time and there's no spelling errors or case mismatch.
```

Comment:
It's a Windows only issue, because the path in Windows is case insensitive. It may cause the poetry regards to different path with different cases.
The spelling error won't happen on a path. If a path is wrong, you cannot get into the folder.

Reply:
I though Windows is case-insensitive 
For spelling error I'm just adding it as a common note there. I think that could be an overlooked error from time to time. If it's not necessary, I can delete that.

Reply:
The case insensitive means, if there is a Windows path called `c:\abc`. Either `c:\abc` or `c:\ABC` can enter it. But the command prompt will keep be what you typed. The two different case path will be regarded as two different path in Poetry, it causes inconsistent problem. But in Linux, if you type `ABC`, you cannot enter a patch `abc`.
Spelling error will get error message from command line in Both Windows and Linux. We should assume user can understand that.

Reply:
Got it. Thank you for your explanation! I will modify it later.